tests/src/JIT/Performance/CodeQuality/BenchmarksGame/spectralnorm/spectralnorm.cs

```diff
@@ -49,13 +50,14 @@ public static void Bench()
                     a += s.Approximate(n);
                 }
             }
-        }
-        double norm = a / (n * Iterations);
-        double expected = 1.274219991;
-        bool valid = Math.Abs(norm - expected) < 1e-4;
-        if (!valid)
-        {
-            throw new Exception("Benchmark failed to validate");
+
+            double norm = a / Iterations;
+            double expected = 1.274219991;
+            bool valid = Math.Abs(norm - expected) < 1e-4;
+            if (!valid)
+            {
+                throw new Exception("Benchmark failed to validate");
+            }
```

Comment:
Are you able to test and verify that this does not fail as a perf test?
Nit: Indentation looks a little off.